spec/data/node_spec.rb

```diff
@@ -366,13 +382,13 @@ def make_tagged_policy(sort_order)
         policy20 = make_tagged_policy(20)
         node.checkin("facts" => { "f1" => "a" })
         node.reload
-        node.tags.should be_empty
         node.policy.should == random_policy
       end
 
       it "should not change when a tag changes" do
```

Comment:
This title would need to be updated, as written: `"should update tags"` or similar.